src/harness/BenchmarkDotNet.Extensions/ValuesGenerator.cs

```diff
@@ -135,5 +137,12 @@ private static string GenerateRandomString(Random random, int minLength, int max
 
             return builder.ToString();
         }
+
+        private static Guid GenerateRandomGuid(Random random)
+        {
+            byte[] bytes = new byte[16];
+            random.NextBytes(bytes);
```

Comment:
thank you for adding `Guid` support and ensuring that we are always going to generate the same data by reusing the `random` instance! 👍